### PR TITLE
Make textarea readonly

### LINKED ISSUE #22: Bug: Can Edit Text area
**Describe the bug**
image

**To Reproduce**
change the text in the text area

**Expected behavior**

unchnageable

**Browser Information**

Google Chrome | 91.0.4472.33 (Official Build) beta (64-bit)
-- | --
Revision | 35d909982aeb034113732460de33d77009059efa-refs/branch-heads/4472@{#659}
Platform | 13904.22.0 (Official Build) beta-channel octopus

**Additional context**
@gosoccerboy5 was this on purpose? I know of  a simple fix for this.

### PR DESCRIPTION
**Pull request by Shluffy**
**Resolves**

Resolves #22 

**Changes**

make textarea readonly
make a new page where you can change the bookmarklet before copying it.

**Tests**

tested locally

### CODE REVIEW

File: change-bookmarklets.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                let bookmarkletsDiv = document.querySelector("#bookmarklets");
+                fetch("https://scratch-bookmarklets.github.io/bookmarklets.json").then(res => res.json()).then(data => {
+                    for (let item in data) {
+                        let newDiv = document.createElement("div");
+                        newDiv.innerHTML = `<h1>${data[item].name}</h1>
+                        <textarea id = "item${item}">${data[item].code}</textarea>`;
+                        bookmarkletsDiv.appendChild(newDiv);
+                        document.querySelector("#item" + item).addEventListener("click", function() {
+                            this.select();
+                        });
```
Comment (gosoccerboy5) [inline change-bookmarklets.html:20]: Remove these lines plz, because auto-selection defeats the purpose

**Review by gosoccerboy5 — CHANGES_REQUESTED** [review]

File: change-bookmarklets.html (hunk 1 above)
Comment (gosoccerboy5) [reply change-bookmarklets.html:20]: Also, I hope you will add a slightly more descriptive title than `Hello! These are bookmarklets you can use to easily do things around the Scratch website.`

Comment (Shluffy) [reply change-bookmarklets.html]: > Also, I hope you will add a slightly more descriptive title than `Hello! These are bookmarklets you can use to easily do things around the Scratch website.`

I will do that in a new pr.

Comment (Shluffy) [reply change-bookmarklets.html]: > Remove these lines plz, because auto-selection defeats the purpose

removed

File: change-bookmarklets.html [hunk 2]
```diff
@@ -14,10 +14,6 @@
                         let newDiv = document.createElement("div");
                         newDiv.innerHTML = `<h1>${data[item].name}</h1>
                         <textarea id = "item${item}">${data[item].code}</textarea>`;
-                        bookmarkletsDiv.appendChild(newDiv);
```
Comment (gosoccerboy5) [inline change-bookmarklets.html]: we should keep this line or else nothing displays

**Review by gosoccerboy5 — CHANGES_REQUESTED** [review]

File: change-bookmarklets.html (hunk 2 above)
Comment (Shluffy) [reply change-bookmarklets.html]: Removed that by accident. readded it.

**Review by gosoccerboy5 — APPROVED** [review]
Perfect! 😄

**Pull request merged**